images/scripts/fedora-rawhide-anaconda-payload.bootstrap

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+set -eux
+
+OUTPUT="$1"
+
+python3 create-anaconda-payload --output=$OUTPUT
```

Comment:
Hmm -- Would you mind renaming create-anaconda-payload to *this* bootstrap script, drop the `--output` argument, and avoid this extra shell wrapper?

Reply:
I see two small problems with the move. 

First is that imports like BOTS_DIR and testvm will not be really possible without some hack to change the python path.

Second is that the seperate script will allow us in the future to reuse this for creating the payload for other fedora versions. 

What do you think?

Reply:
Sure. Not really important.